ui/app/AppLayouts/Profile/views/wallet/ManageTokensView.qml

```diff
@@ -20,6 +20,7 @@ import AppLayouts.Wallet.panels 1.0
 Item {
     id: root
 
+    required property int tokenListUpdatedAt
```

Comment:
I'd make this a `double` everywhere in QML in order not to overflow the UNIX timestamp